protocol/transactions.go

```diff
@@ -0,0 +1,57 @@
+package protocol
+
+import (
+	"github.com/kaspanet/kaspad/protocol/peer"
+	"github.com/kaspanet/kaspad/util"
+	"github.com/kaspanet/kaspad/util/daghash"
+	"github.com/kaspanet/kaspad/wire"
+	"github.com/pkg/errors"
+	"time"
+)
+
+// AddTransaction adds transaction to the mempool and propagates it.
+func (m *Manager) AddTransaction(tx *util.Tx) error {
+	m.transactionsToRebroadcastLock.Lock()
+	defer m.transactionsToRebroadcastLock.Unlock()
+	acceptedTxs, err := m.txPool.ProcessTransaction(tx, false, 0)
```

Comment:
Here too - rename to something that makes it clear these are transactions accepted by mempool